### PR TITLE
dm/test: fix tiup upgrade test

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
fix the tiup upgrade test failure problem

### What is changed and how it works?
set start-relay check count in old version dm from 2(new) to 1(original)

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: dm/tests/tiup/lib.sh [hunk 1]
```diff
@@ -269,6 +269,10 @@ function ensure_start_relay() {
 		# always use CUR_VER, because we might use tiup mirror in previous steps.
 		tiup dmctl:$CUR_VER --master-addr=master1:8261 operate-source show -s mysql-replica-01 >$dmctl_log 2>&1
 		worker=$(grep "worker" $dmctl_log | awk -F'"' '{ print $4 }')
-		run_dmctl_with_retry $CUR_VER "start-relay -s mysql-replica-01 $worker" "\"result\": true" 2
+		if [[ "$PRE_VER" == "v2.0.2" ]] || [[ "$PRE_VER" == "v2.0.3" ]] || [[ "$PRE_VER" == "v2.0.4" ]] || [[ "$PRE_VER" == "v2.0.5" ]] || [[ "$PRE_VER" == "v2.0.6" ]]; then
```
Comment (lance6716) [inline dm/tests/tiup/lib.sh:272]: forget to add v2.0.7 in `previous_v2` :joy: maybe we can only test important versions

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by lance6716 — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**